tracing-core/src/callsite.rs

```diff
@@ -202,7 +208,10 @@ impl<T> Registration<T> {
 
 impl fmt::Debug for Registration {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_struct("Registration").finish()
+        f.debug_struct("Registration")
+            .field("callsite", &format_args!("{:p}", self.callsite))
+            .field("next", &format_args!("{:p}", self.next.load(Ordering::Acquire))
```

Comment:
agh, my bad: my suggested code was missing a closing paren here, so this doesn't compile:
```suggestion
            .field("next", &format_args!("{:p}", self.next.load(Ordering::Acquire)))
```